snap/snapcraft.yaml

```diff
@@ -326,6 +326,12 @@ apps:
       SecretService_TokenPath: $SNAP_DATA/secrets/edgex-security-proxy-setup/secrets-token.json
       SecretService_CACertPath: $SNAP_DATA/secrets/ca/ca.pem
     plugs: [home, removable-media, network]
+  secrets-config:
+    adapter: none
+    command: bin/secrets-config
```

Comment:
The README says that `--confdir` is optional, however since you're already copying security-proxy-setup's configuration file to `$SNAP_DATA/config/secrets-config/res`, shouldn't you just specify it here? That said, see my global comments about the problems I ran into while trying to test your command from the snap.

Reply:
I am somewhat lacking context here.  Are you saying that all of the other snapcraft defintions explicitly point at --confdir that this one isn't?

Reply:
Changed to `command: bin/secrets-config -confdir $SNAP_DATA/config/secrets-config/res`